src/DesignKit.tsx

```diff
@@ -519,34 +527,136 @@ const SelectSection: React.FC = () => {
   );
 };
 
-const ThemeSection: React.FC = () => (
-  <ComponentSection id="Theme" title="Theme">
-    <AntDCard>
-      <p>
-        <code>{'<UIProvider>'}</code> is part of the UI kit, it is responsible for handling all
-        UI/theme related state, such as dark/light theme setup. It takes an optional{' '}
-        <code>{'branding'}</code> prop for adjusting branding specific theme/colors.
-      </p>
-      <p>
-        Besides the <code>{'<UIProvider>'}</code>, there are a few other helpers that can be used
-        from withing the UI kit.
+const ThemeSection: React.FC = () => {
+  const { themeState } = useThemeState();
+  const isDarkMode = themeState.themeIsDark;
+  const baseTheme: Theme = isDarkMode ? themeDarkDetermined : themeLightDetermined;
+
+  const colorVariations = [
+    { color: baseTheme.statusActive, name: Status.Active },
+    { color: baseTheme.statusCritical, name: Status.Critical },
+    { color: baseTheme.statusPendingWeak, name: Status.PendingWeak },
+    { color: baseTheme.statusSuccess, name: Status.Success },
+    { color: baseTheme.statusWarning, name: Status.Warning },
+  ];
+
+  const themes = colorVariations.map((variation) => ({
+    theme: {
+      ...themeLightDetermined,
+      backgroundOnStrong: variation.color,
+      brand: variation.color,
+      stageBorder: variation.color,
+      statusSuccess: variation.color,
+      surface: variation.color,
+    },
+    variation,
+  }));
+
+  const themeVariations = themes.map((themeVariation) => {
```

Comment:
Can you add an example of one of the components that's using `getContainer` like `Drawer` or `Toast` to ensure those work correctly with nested `<UIProvider>`s?

Reply:
Added both a Toast and Drawer example